Microsoft.NET.Build.Containers/Registry.cs

```diff
@@ -106,6 +109,19 @@ public Registry(Uri baseUri)
         };
     }
 
+    private Image GenerateNewEmptyImage()
+    {
+        var config = new JsonObject(new KeyValuePair<string, JsonNode?>[]{
+            new("rootfs", new JsonObject(new KeyValuePair<string, JsonNode?>[]{
+                new("type", "layers"),
+                new("diff_ids", new JsonArray())
+            })),
+            new ("config", new JsonObject()),
+            new ("history", new JsonArray())
+        });
```

Comment:
I'm actually unsure how much of this is strictly required - the main one was `rootfs.diff_ids` - we do direct indexing in `Image.cs`, perhaps if we changed that to be more safe navigation then we wouldn't need to create a config structure at all?